idaes/models_extra/column_models/condenser.py

```diff
@@ -243,9 +243,27 @@ def build(self):
 
         self._make_ports()
 
+        @self.Expression(doc="Split fraction calculation calculations")
+        def reflux_split_fraction(self):
+            return self.reflux_ratio * (1 + self.reflux_ratio) ** -1
+
         if self.config.condenser_type == CondenserType.totalCondenser:
 
-            self._make_splits_total_condenser()
+            make_phase_split(
+                self.control_volume,
+                port=self.reflux,
+                phase=self._liquid_set,
+                side_sf=(self.reflux_split_fraction),
+                equipmentType=CondenserType.totalCondenser,
+            )
+
+            make_phase_split(
+                self.control_volume,
+                port=self.distillate,
+                phase=self._liquid_set,
+                side_sf=(1 - self.reflux_split_fraction),
+                equipmentType=CondenserType.totalCondenser,
+            )
 
             if self.config.temperature_spec == TemperatureSpec.atBubblePoint:
```

Comment:
@agarciadiego A note for the future. This needed to be changed to support the newer version of the square flash formulation where we removed the need for bubble and dew point. The change I made (and that is sitting on my computer) is to replace this instead with a sum{k_i*x_i = 1} constraint.

Reply:
added to issue #96